addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxServerHandler.java

```diff
@@ -444,6 +445,15 @@ private String decode(String raw) {
             }
         }
 
+        private String encode(String raw) {
+            try {
+                return URLEncoder.encode(raw, "UTF-8");
```

Comment:
I would prefer to use `StandardCharsets.UTF_8.name()` [see](https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html#UTF_8) 

And you can import the `UTF_8` to make it a little more compact.